backend/main.py

```diff
@@ -113,6 +113,36 @@ def new_game():
   return jsonify(firebase.put('/games', game, put_data))
 
 
+@app.route('/joingame', methods=['POST'])
+def join_game():
+  request_data = request.get_json()
+  game_to_join = request_data['gameId']
+  player_to_add_to_game = request_data['playerId']
+  name = request_data.get('name', '')
+  need_gun = request_data.get('needGun', False)
+  profile_image_url = request_data.get('profileImageUrl', '')
+  start_as_zombie = request_data.get('startAsZombie', False)
+  volunteer = request_data.get('volunteer', False)
+  be_secret_zombie = request_data.get('beSecretZombie', False)
+  user_id = request.headers['userId']
+
+  player_info = {
+    'gameId': game_to_join
+  }
+  firebase.put('/users/' + user_id + '/players', player_to_add_to_game, player_info)
```

Comment:
Is that's the intent, there's no need for a separate method. I just think the method name is misleading. It certainly mislead me. joinGame sounds like an existing player is joining an existing game. If the intent is to create a new player as part of an existing game, I think something like addPlayer is a more clear name and signals better what's being done. Now that I know what this method actually does, it's just a matter of semantics (and misleading others). If the name isn't changed, there should definitely be a comment making it clear what's going on.